<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDlg</class>
 <widget class="QDialog" name="SearchDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>443</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/search_bg.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="friendTabButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/transparent.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>添加好友</string>
   </property>
   <property name="icon">
    <iconset resource="../pixmap.qrc">
     <normaloff>:/pixmap/image/addFriendIco.png</normaloff>:/pixmap/image/addFriendIco.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="roomTabButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>120</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/transparent.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>搜索聊天室</string>
   </property>
   <property name="icon">
    <iconset resource="../pixmap.qrc">
     <normaloff>:/pixmap/image/searchRoomIco.png</normaloff>:/pixmap/image/searchRoomIco.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="friendTab" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>443</width>
     <height>260</height>
    </rect>
   </property>
   <widget class="QPushButton" name="sarchUserButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>99</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pixmap/image/searchButton.png);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFriendButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
    </property>
    <property name="text">
     <string>添加好友</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeFriendTabButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>234</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pixmap/image/pw.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QListView" name="searchUserlistView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pixmap/image/transparent.png);
border-width: 1px;
   border-style: solid;
   
border-color: rgba(255, 124, 107, 150);
</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="roomTab" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>451</width>
     <height>260</height>
    </rect>
   </property>
   <widget class="QPushButton" name="closeRoomTabButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="joinRoomButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
    </property>
    <property name="text">
     <string>加入</string>
    </property>
   </widget>
   <widget class="QListView" name="searchRoomListView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pixmap/image/transparent.png);
border-width: 1px;
   border-style: solid;
   
border-color: rgba(255, 124, 107, 150);
</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../pixmap.qrc"/>
 </resources>
 <connections/>
</ui>
